src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrCss = sourceString
+    .split(';')
+    .map((rule) => rule.trim())
+    .filter(Boolean);
+
+  const obgElm = {};
```

Comment:
There is a typo in the variable name `obgElm`. It should be `objElm` to correctly represent 'object'.